src/averageAges.js

```diff
@@ -14,12 +14,20 @@
  *
  * @return {number}
  */
+function averageAge(age) {
+  return age.reduce((ages, { died, born }) => {
+    return ages + (died - born);
+  }, 0) / age.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mens = (!century)
+    ? people.filter(({ sex }) => sex === 'm')
+    : people.filter(({ sex, died }) => (
+      sex === 'm' && Math.ceil(died / 100) === century
+    ));
```

Comment:
1. Use correct English
2. Let's use more reliable check here, because century, theoretically, can have falsy value (0).

```suggestion
  const men = (century === undefined)
    ? people.filter(({ sex }) => sex === 'm')
    : people.filter(({ sex, died }) => (
      sex === 'm' && Math.ceil(died / 100) === century
    ));
```